apps/codelab/src/app/components/buttons-nav-bar/buttons-nav-bar.component.html

```diff
@@ -4,4 +4,6 @@
   <codelab-menu-github-widget></codelab-menu-github-widget>
   <feedback-widget></feedback-widget>
   <codelab-menu-shortcut-widget></codelab-menu-shortcut-widget>
+  <codelab-logout-widget *access="accessService.isAuthorized$ | async"></codelab-logout-widget>
```

Comment:
This directive does not seem to be very useful, looks like it's just a clone of ngIf, with some stuff missing.
What would be cool to have is something like `*isAuthorized` (or maybe `*isloggedIn` would be even better)